benchmarks/templates/benchmarks/base_v2.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
```

Comment:
I would still update `base.html` rather than introduce a new file. Let's not do custom versioning and just update everything directly